### PR TITLE
Brownie support

### PR DESCRIPTION
**Pull request by tammam1998**
Basic support for syncing brownie metadata.

### CODE REVIEW

**Comment by antoinedc** [comment]
Thanks for the PR! The code looks good to me, I still need to test it locally for a couple of use cases. If all goes well I will merge it tomorrow at the latest!

**Comment by antoinedc** [comment]
There seems to be one issue: artifacts are not directly in the deployment folder, when I run `brownie run token.py` (with `token.py` being the deployment file), it creates a `dev` folder in `deployments`, next to `map.json`, that contains the artifact.

I'm not sure how it is on your end, but for me this doesn't work because of the hardcoded `build/deployment` path in in the code. Any idea on how this extra folder is generated? We need to have a way to reliably get to the artifact.

An option might be to use the `map.json` file, but I'm not entirely sure either on how to get the correct info from it if multiple networks are used for example.

Feel free to ping me on Discord to talk about this further if that's easier!

File: bin/index.js [hunk 1]
```diff
@@ -236,6 +257,60 @@ function getTruffleArtifact(artifactsDir, fileName) {
...
+    if (!dir) {
+        console.log('Please specify a directory to watch.');
+        return;
+    }
+    // Might need to add a way to specify which deployment folder to watch
+    const artifactsDir = path.format({
+        dir: dir,
+        base: "build/deployments"
+    }); 
+    const watcher = chokidar.watch('.', { cwd: artifactsDir })
```
Comment (antoinedc) [inline bin/index.js]: Replace with `const watcher = chokidar.watch('./**/*.json', { cwd: artifactsDir, ignored: 'map.json' })`. I somehow had a non json file in the folder that caused an error, and `map.json` shouldn't be useful to parse.

**Comment by antoinedc** [comment]
Actually nevermind, I had another issue on my end and it's working properly :) I've added just one comment because of some minor error thrown in the console. Will merge when you've updated the code!

File: bin/index.js (hunk 1 above)
Comment (tammam1998) [reply bin/index.js]: done!

**Comment by tammam1998** [comment]
I see, thanks for looking into this. I addressed the comment!

**Comment by antoinedc** [comment]
Perfect, thanks!

**Pull request merged**